Dockerfile

```diff
@@ -0,0 +1,112 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright 2020 (C) Olliver Schinagl <oliver@schinagl.nl>
+
+# hadolint ignore=DL3007  latest is the latest stable for alpine
+FROM registry.hub.docker.com/library/alpine:latest AS builder
```

Comment:
I'm very tempted to recommend switching to `debian:buster-slim`.  
We have a couple of minor compatibility issues with Alpine.
1. ClamOnAcc doesn't support Alpine.  I don't know if anyone will ask for ClamOnAcc in a docker image in the future... Probably not... so maybe this doesn't matter.
2. Alpine appears to have some valgrind/helgrind issues in our test system when using iconv to convert from japanese (Shift_JIS) to utf-8.  I think it's fairly benign but we have it marked as an expected failure in our internal QA test suite.

A couple other teams in my org also use `debian:buster-slim`, which is why I thought of it.  I otherwise like Alpine.  I don't care too much at this point, and would be happy enough if you wanna stick with Alpine.

Reply:
how about, MVP, keep alpine for now, and then add both in the future? Just to keep the size of this MR down?

Reply:
That that you mention it, I'm pretty sure inotify/fanotify isn't available for docker containers.  So even if we could build ClamOnAcc, it wouldn't work.  Most of our QA pipeline runs in docker for testing on Linux and we have an incomplete work-in-progress for a separate pipeline that tests on a couple VM's instead for ClamOnAcc.  Sadly we had some turnover, so that work went on the backburner some time ago, and ClamOnAcc testing has since still been mostly manual.  😢  I have a pretty solid idea how to finish it but lack the time to work on it.